src/mesh/README.rst

```diff
@@ -20,7 +20,7 @@ Commands to create an application
 ::
 
 	Command
-		az mesh app create: Create an Service Fabric Mesh application.
+		az mesh deployment create: Create an Service Fabric Mesh application.
 
```

Comment:
The following examples need to be updated. Also, paste the latest output from az mesh deployment create -h in case something has been changed